public/employers/disney.yml

```diff
@@ -0,0 +1,48 @@
+name: The Walt Disney Company
+aliases:
+  - Walt Disney Productions
+  - Walt Disney Studio
+shortName: Disney
+ticker: DIS
+wiki: The Walt Disney Company
+image: disney.svg
+officialWebsite: https://thewaltdisneycompany.com/about/
+industries:
+  - entertainment
```

Comment:
Their main industry is entertainment, but they are also in hospitality/lodging, media, and some retail like apparel or clothing.

Reply:
Yeah this is a tough call. Wikipedia only puts them as media and entertainment. But I think apparel and hospitality would be two others to include as well.